src/Game.js

```diff
@@ -14,87 +14,98 @@ function Game () {
 		])
 	)
 
-	const handleEnemyEndOfTurn = (playerName) => {
-		console.log('Your turn ' + playerName) // TODO: Remove these
-
+	function handleEnemyEndOfTurn (playerName) {
 		setWhoseTurn(playerName)
 	}
 
-	const handlePlayerHasLost = (playerName) => {
-		const winner =
-			playerName === PLAYER_ONE_NAME ? PLAYER_TWO_NAME : PLAYER_ONE_NAME
+	function handlePlayerHasLost (playerName) {
+		const winner = playerName === PLAYER_ONE_NAME ? PLAYER_TWO_NAME : PLAYER_ONE_NAME
 
-		console.log('You lost ' + playerName)
-		console.log('Congratulations ' + winner)
-		setGameOver(true)
+		setWhoWon(winner)
 	}
 
-	const handlePlayerShipPlacement = (playerName) => {
-		const whoseTurn =
-			playerName === PLAYER_ONE_NAME ? PLAYER_TWO_NAME : PLAYER_ONE_NAME
+	function handlePlayerShipPlacement (playerName) {
+		const whoseTurn = playerName === PLAYER_ONE_NAME ? PLAYER_TWO_NAME : PLAYER_ONE_NAME
 		const shipsOnBoardCopy = new Map(shipsOnBoard)
 
 		shipsOnBoardCopy.set(playerName, true)
 		setWhoseTurn(whoseTurn)
 		setShipsOnBoard(shipsOnBoardCopy)
 	}
 
-	return <div className='game'>{renderPlayers()}</div>
+	function renderAPlayer (name, hideBoard, showShipOnBoard, unclickableBoard = false) {
+		let customStyle = hideBoard ? 'hide-board' : ''
+		customStyle = unclickableBoard ? `${customStyle} unclickable-board` : customStyle
+
+		return (
+			<Player
+				key={name}
+				name={name}
+				customStyle={customStyle}
+				showShipOnBoard={showShipOnBoard}
+				onPlayerHasLost={handlePlayerHasLost}
+				onEnemyEndOfTurn={handleEnemyEndOfTurn}
+				onPlayerShipPlacement={handlePlayerShipPlacement}
+			/>
+		)
+	}
+
+	function renderGameInfo (info, onWin = false) {
+		if (!onWin) {
```

Comment:
I'm not quite sure why you split this into two different if-branches.

You could simply do:
```
<div key="info" className="split-child-container">
 <p className="text-center">{info}</p>
 {onWin && 
   <button className="reset-button" onClick={() => window.location.reload()}>
	Play again!
   </button>}
</div>
```

Reply:
You're able to combine code with html, so try to do it as often as possible (where it makes sense) to reduce redundancy